github-notify.go

```diff
@@ -34,10 +36,22 @@ func retrieveNotifications(client *github.Client) []github.Notification {
 	opts := &github.NotificationListOptions{All: false, Participating: true}
 	var results []github.Notification
 	notes, _, _ := client.Activity.ListNotifications(opts)
+	// Oldest notifications first
+	notes = reverseNotifications(notes)
+
+	// Clean up notRead
+	for i := len(notRead) - 1; i >= 0; i-- {
+		if(!hasNotification(notes, notRead[i])) {
+			notRead = append(notRead[:i], notRead[i+1:]...)
+		}
+	}
+
 	for _, note := range notes {
```

Comment:
I was just thinking about this, why reverse the notes in a special function, if you can just loop over it in reverse? #saveALoop